generators/spring-controller/templates/src/main/java/package/web/rest/_Controller.java

```diff
@@ -0,0 +1,35 @@
+<%#
+ Copyright 2013-2017 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see http://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.web.rest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * <%= controllerClass %> controller
+ */
+@RestController
+@RequestMapping("/api/<%= apiPrefix %>")
```

Comment:
Maybe we should also generate a unique URL? Here you would get error when you generate 2 controllers in a row. How about "/api/<%= apiPrefix %>-<%=controllerClass%>"

Reply:
Maybe use Lodash to use a dasherized version of "controllerClass"

Reply:
apiPrefix = kebabCase(controllerClass)

Reply:
the issue with KebabCase is that you would keep upper case letters -> maybe do a lower case after?

Reply:
no, it doesn't, I tried already...

```
/**
 * AnotherTest controller
 */
@RestController
@RequestMapping("/api/another-test")
public class AnotherTestController {
    //...
}
```

Reply:
I'm quite surprised, good news then, let's do it!